tools/python/boutiques/invocationSchemaHandler.py

```diff
@@ -3,7 +3,12 @@
 # A script for handling invocation schema generation and validation for Boutiques application descriptors
 # Requires jsonschema 2.5
 
-import json, jsonschema as jsa, argparse, os, sys
+import json
+import jsonschema
```

Comment:
Take a look here for your issue (https://github.com/boutiques/boutiques/issues/105). You don not have anymore `jsa` defined, you should use `jsonschema` instead.